sdk/timeseriesinsights/Azure.Iot.TimeSeriesInsights/src/Customized/Models/InstancesOperationError.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Azure.Core;
+
+namespace Azure.Iot.TimeSeriesInsights
+{
+    /// <summary>
+    /// A particular API error with an error code and a message.
```

Comment:
Please comment why it was renamed (but not in a public comment).

E.G. <https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/digitaltwins/Azure.DigitalTwins.Core/src/Customized/Models/CreateModelsOptions.cs#L11>

Reply:
btw, what I meant by not in a public comment is not as a doc comment that will show up in tooltips and in web docs

Reply:
Yup. I added a non-public comment like you suggested on lines 14 through 16. Does that work?

Reply:
Looks good.